types/react-native/index.d.ts

```diff
@@ -9470,8 +9486,7 @@ interface NativeModulesStatic {
  * <code>const MyModule = NativeModules.ModuleName</code>
  */
 export const NativeModules: NativeModulesStatic;
-export const Platform: PlatformStatic;
-export const PlatformIOS: PlatformIOSStatic;
```

Comment:
This is going to break some code… but not sure how to best deprecate that otherwise 🤔 I guess readers will find this PR soon enough, when things break 🤷‍♂

Reply:
> I guess readers will find this PR soon enough, when things break

Sure, btw https://github.com/search?q=PlatformIOSStatic&type=Code

35 code results in the whole github, i think it's not used (only 0.001% of all RN users)